test-cypress/cypress/commands/server.ts

```diff
@@ -0,0 +1,85 @@
+// load the global Cypress types
+/// <reference types="cypress" />
+export const DEFAULT_DATE = new Date(2021, 1, 16, 22);
+import url = require("url");
+
+export function apiPath() : string {
+   return Cypress.env('cacophony-api-server');
+}  
+
+export function v1ApiPath(page: string, queryParams : any = {}) : string {
+   const urlpage = url.format({
+      pathname:`/api/v1/${page}`,
+      query: queryParams
+    });
+   return `${Cypress.env('cacophony-api-server')}${urlpage }`;
+}
+
+interface apiCreds {
+   name: string,
+   headers: {
+     'authorization': any
+   },
+   jwt: string,
+   id: number
+}
+
+export function getCreds(username: string) : apiCreds {
+   return Cypress.env('testCreds')[username];
+}
+
+export function saveCreds(response: Cypress.Response, name: string, id = 0) {
+   const creds = {
```

Comment:
In es6, you don't need to do `name: name`, `id: id` - you can just write { name, id, ... } etc.

Reply:
Thanks for the tip.   Soo nice :)